docs/pages/configuration/pipelines/functions/get_config_value.mdx

```diff
@@ -2,3 +2,17 @@
 title: Function get_config_value
 sidebar_label: get_config_value
 ---
+
+Description needed here
+
+## Arguments
+
+
+## Usage
+
+```
+get_config_value [configValue1]
+```
+
+
+## Flags
```

Comment:
Thoughts on removing ##Flags since it doesn't have any?

Reply:
I added "None" here but will ask Fabian what he prefers.